gilgo_padavick_final_project/deliverables/deliverable_1/README.md

```diff
@@ -0,0 +1,23 @@
+## Project Deliverable 1
```

Comment:
Question 1 of the analysis: Can we classify specific market behavior as a pump?
Answer: The identification of a pump can be easily seen through historical data, with daily charts showing what appears to be a blip in the price when viewing it from the 1Y or 2Y view. To get more specific on how to classify them as they start and not just after the fact, features in the data such as average volume per minute and percentage of price change give us a tip about if the stock is being either pumped or dumped as well. Pairing that in conjunction with an increase in Twitter activity is what we are trying to quantify, and once we are able to work through the Twitter API issues, it will be easier to calculate what percentage changes constitute a "pump" and help our model make clear decisions.

Question 3: What is the impact to new investors, positive or negative, that trade in reaction to these tweets?
Answer: Again, working through the Twitter API will give us an idea about where the masses have peaked in there mentions on twitter about the stock, also telling us where to calculate the price change from when looking at the total impact. In general though, there are stocks such as Kodak, SHIP, and Nikola that all dropped 80%+ from the peaks of their pump prices, so while we know that is a large impact on the potential trader buying at the highest price (AKA the "bagholder"), we aren't able to quantify as a number the impact to the average trader. We will seek to have this answered as we move forward in the project.